src/libraries/System.Diagnostics.DiagnosticSource/src/System.Diagnostics.DiagnosticSource.csproj

```diff
@@ -61,12 +60,9 @@
       <Link>Common\System\LocalAppContextSwitches.Common.cs</Link>
     </Compile>
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetFramework)' != 'net45'">
+  <ItemGroup>
     <Compile Include="System\Diagnostics\Activity.Current.net46.cs" />
```

Comment:
nit: should we rename this file ?

Reply:
Yes good point. Let's do that another time to not reset the PR.